app/js/view/details_view.js

```diff
@@ -35,14 +39,75 @@ export default class DetailsView extends View {
     });
   }
 
+  filterNonAffectedApps(apps, filters) {
+    return apps.filter(app => {
+      return filters.reduce((prev, cur) => {
+        return prev || cur.test(app.manifestURL);
+      }, false);
+    });
+  }
+
+  populateAffectedApps(details) {
+    Promise.all([
+      AppsHelper.getAllApps(),
+      ManifestHelper.getManifest(details.manifestURL)
+    ])
+
+    .then(results => {
+      var apps = results[0];
+      var manifest = results[1];
+      var filters = manifest.customizations.map(customization => {
+        if (!customization.filter) {
+          console.warn('No filter provided in customization', customization);
+          customization.filter = '';
+        }
+        return new RegExp(customization.filter);
+      });
+
+      // Test the current manifest against each customization filter
+      // and mark the app as affected if it matches at least one.
+      var filteredApps = this.filterNonAffectedApps(apps, filters);
+
+      var affectedApps;
+      if (apps.length === filteredApps.length) {
+        affectedApps = 'All applications.';
+      } else {
+        affectedApps = filteredApps.reduce((prev, cur) => {
+          return (prev ? prev + ', ' : '') + cur.manifest.name;
+        }, null);
+      }
+
+      this.affectedApps.textContent = affectedApps || 'None';
+    })
+
+    .catch((err) => {
+      console.warn('Could not populate affected apps', err);
+      // Hide affected apps section when undetermined.
+      this.addonSection.hidden = true;
+    });
+  }
+
   show(details) {
     this.details = details;
     this.titleElement.textContent = capitalize(details.name);
     IconHelper.setImage(this.iconElement, details.icon);
     this.nameElement.textContent = capitalize(details.name);
     this.authorElement.textContent = details.author;
-    this.installButton.textContent = details.installed ? 'Open' : 'Install';
     this.fullDescription.textContent = details.description;
+    this.installButton.textContent = details.installed ? 'Open' : 'Install';
+    this.installButton.disabled = false;
+    this.addonSection.hidden = true;
+
+    // Addons need the affected apps section, and no Open button.
```

Comment:
Yes, but this raises an interesting point. Maybe we should provide a shortcut or option to enable/disable the add-on from here.